Add information for health checks to data lake documentation (#118787)
Azure/azure-content: articles/healthcare-apis/dicom/dicom-data-lake.md

## Access tiers

You can manage costs for imaging data stored by the DICOM service by using Azure Storage access tiers for the data lake storage account. The DICOM service only supports online access tiers (either hot, cool, or cold), and can retrieve imaging data in those tiers immediately. The hot tier is the best choice for data that is in active use. The cool or cold tier is ideal for data that is accessed less frequently but still must be available for reading and writing.

To learn more about access tiers, including cost tradeoffs and best practices, see [Azure Storage access tiers](/azure/storage/blobs/access-tiers-overview)

## Health check

> The DICOM service will write a small file to the data lake every 30 seconds, following the [Data Contract](#data-contracts), to ensure it maintains access. Making any changes to files stored under the `healthCheck` sub-directory could result in incorrect status of the health check.

Reviewer: ```suggestion
The DICOM service writes a small file to the data lake every 30 seconds, following the [Data Contract](#data-contracts) to ensure it maintains access. Making any changes to files stored under the `healthCheck` sub-directory might result in incorrect status of the health check.
```

> If there is an issue with access, status and details will be displayed by [Azure Resource Health](../../service-health/overview.md). It will specify if any action is required to restore access, for example re-instating a role to the DICOM service's identity.

Reviewer: ```suggestion
If there is an issue with access, status and details are displayed by [Azure Resource Health](../../service-health/overview.md). Azure Resource Health specifies if any action is required to restore access, for example reinstating a role to the DICOM service's identity.
```
